src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -49,16 +52,23 @@
   StorageHelperTest.class,
   InstanceRelationshipsTest.class,
   ReferenceTablesTest.class,
+  ItemDamagedStatusAPITest.class,
+  ItemDamagedStatusAPIUnitTest.class
 })
-@SuppressWarnings("squid:S1118")  // suppress "Utility classes should not have public constructors"
+@SuppressWarnings("squid:S1118")
+// suppress "Utility classes should not have public constructors"
 public class StorageTestSuite {
   public static final String TENANT_ID = "test_tenant";
 
   private static Vertx vertx;
   private static int port;
 
-  public static URL storageUrl(String path) throws MalformedURLException {
-    return new URL("http", "localhost", port, path);
+  public static URL storageUrl(String path) {
+    try {
+      return new URL("http", "localhost", port, path);
+    } catch (MalformedURLException e) {
+      throw new RuntimeException(e);
```

Comment:
Why is this being converted to a runtime exception? Just so the checked exception can be avoided in calling methods?

Reply:
Yes it is, In case It's used only in tests and an URL is not created by User(any typos we can avoid) I don't think it's necessary to keep this checked exception because it brings much worries for developers